aten/src/ATen/core/boxing/impl/boxing.h

```diff
@@ -262,9 +304,11 @@ struct BoxedKernelWrapper<
     torch::jit::push(stack, std::forward<Args>(args)...);
 
     (*boxed_kernel_func)(functor, opHandle, &stack);
-    TORCH_INTERNAL_ASSERT_DEBUG_ONLY(
```

Comment:
Why TORCH_CHECK? Do you think this could happen when it is not a PyTorch bug but a user error?

Reply:
yeah, I mean all these boxed kernels are on the honor system, right? any time a user-supplied kernel pushes the wrong number of return values, it's going to end up here. (E.g. this happened to Richard [here](https://gist.github.com/zou3519/90428b51bf4b51285ac9c06de5c8f98e) - in this case it turned out he was right and the boxing code was wrong, but the effect would have been the same in the reverse case)